src/tags/object/Ranker.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * libraries
+ */
+import React from 'react';
+import { inject, observer } from 'mobx-react';
+import { flow, types } from 'mobx-state-tree';
+
+import Registry from '../../core/Registry';
+import { AnnotationMixin } from '../../mixins/AnnotationMixin';
+import ProcessAttrsMixin from '../../mixins/ProcessAttrs';
+import Base from './Base';
+import { parseValue } from '../../utils/data';
+
+/**
+ * components
+ */
+import RankerBoard from '../../components/RankerBoard/RankerBoard';
+import { getData } from '../../components/RankerBoard/createData';
+
+/**
+ * @name RankerBoard
+ * @meta_title RankerBoard Tag displays items that can be dragged and dropped between columns
+ * @meta_description Customize Label Studio by displaying and sorting results for machine learning and data science projects.
+ * @param {string} value Data field value containing JSON type for RankerBoard
+ * @param {string} [valueType] Value to define the data type in RankerBoard
+ */
+const Model = types
+  .model({
+    type: 'rankerboard',
+    value: types.maybeNull(types.string),
+    _value: types.frozen([]),
+    valuetype: types.optional(types.string, 'json'),
+    /**
+     * rank: 1 column, reorder to rank
+     * select: 2 columns, drag results to second column to select
+     */
+    _mode: types.optional(types.enumeration(['rank', 'select']), 'select'),
```

Comment:
It should not be underscored — this props is reflecting the actual optional attribute of tag an will be used as `<Ranker name="search" value="$results" mode="select" />`.
And it would be cool to add this to examples!